introduction.md

```diff
@@ -1,3 +1,11 @@
 Lorem ipsum dolor sit amet, consectetur adipisicing elit. Velit soluta, atque repellendus consequuntur at minima ex vitae unde tempora harum sapiente rerum eos deleniti ullam obcaecati, laudantium libero dolore. Saepe?
 
-My favorite Code School path is JavaScript
\ No newline at end of file
+My favorite Code School path is JavaScript
+
+My "deadly_skills" are
+
+* Javascript
+* Git
+* HTML & CSS
+* PHP
+* Python
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?